packages/Pill/src/Pill.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import PropTypes from "prop-types";
+import tokens from "@paprika/tokens";
+import RawButton from "@paprika/raw-button";
+import pillStyles from "./Pill.styles";
+
+export const tokenPillColors = [
+  tokens.color.black,
+  tokens.color.blackLighten60,
+  tokens.color.blue,
+  tokens.color.greenDarken10,
+  tokens.color.orangeDarken10,
+];
+export const severityPillColors = ["noRisk", "lowRisk", "mediumRisk", "highRisk"];
+export const pillSizes = ["small", "medium"];
```

Comment:
I think that this will be a common enough use case to define another "set", like:
https://github.com/acl-services/paprika/blob/master/packages/helpers/src/customPropTypes.js#L10
But maybe `ShirtSizes.LIMITED`?